### DISCUSSION TITLE
stand is detecting yim as presence spoofing

### DISCUSSION THREAD

**Discussion by lonelybud** [YimMenu/YimMenu#2347] [General]
![image (1)](https://github.com/YimMenu/YimMenu/assets/139460769/afedce5c-5b00-4463-91e2-7e02767a6993)

This photo is from one of my friend using stand latest version. He shared this snapshot. 

i checked. only god and spectate spoof was checked under spoofing in yim menu. Also i am not spoofing host token.

Is it a concern? Have anyone seen this before?

**Comment by pierrelasse** [comment]
+1 Yep, same for me. I always get detected for it. Even with nothing on.

**Comment by pierrelasse** [comment]
I got detected with BLOCK RID JOIN off!
[settings.json](https://github.com/YimMenu/YimMenu/files/13259895/settings.json)

**Reply by lonelybud** [reply]
thanks for update

**Reply by Yimura** [reply]
@pierrelasse do you have join redirect enabled for if a certain player comes online? Also could you try to disable relay connections?

**Reply by pierrelasse** [reply]
I have join redirect on yes. Do the target players have to be online?

**Reply by Yimura** [reply]
Yes they do, join redirect would behave the same way as block RID joining. They both make use of spoofing your presence.

**Reply by pierrelasse** [reply]
I just noticed that i don't have join redirect on either due to it being reset by the menu or me disabling it.

Also player.json is taking a lot of storage (400KB). Would it make sense to use numbers or something smaller as keys and translate them back and forth to save storage?
Ex.
```json
"1016356": {
        "block_join": false,
        "block_join_reason": 1,
        "command_access_level": null,
        "custom_infraction_reason": "",
        "infractions": [
            8
        ],
        "is_modder": true,
        "is_trusted": false,
        "join_redirect": false,
        "join_redirect_preference": 1,
        "name": "marci---HUN",
        "notes": "",
        "notify_online": false,
        "rockstar_id": 1016356
    }
```
```json
"1016356": {
        "1": false,
        "2": 1,
        "3": null,
        "4": "",
        "5": [
            8
        ],
        "6": true,
        "7": false,
        "8": false,
        "9": 1,
        "10": "marci---HUN",
        "11": "",
        "12": false,
        "13": 1016356
    }
```
Anything in that direction maybe. Not 1:1.
And would it make sense to not set things that are the default value? The json ready probably lets you get a key and set a default value if not set right?

**Reply by pierrelasse** [reply]
Nice story bro
